src/transport/raw/PeerAddress.h

```diff
@@ -167,6 +170,11 @@ class PeerAddress
 #endif
                 snprintf(buf, bufSize, "TCP:[%s%s]:%d", ip_addr, interface, mPort);
             break;
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
```

Comment:
This should probably not be conditional.

Reply:
I will fix it